bindings/matrix-sdk-ffi/src/client_builder.rs

```diff
@@ -27,6 +27,44 @@ enum HomeserverConfig {
     ServerNameOrUrl(String),
 }
 
+#[derive(uniffi::Enum)]
+pub enum BackupDownloadStrategy {
+    /// Automatically download all room keys from the backup when the backup
+    /// recovery key has been received. The backup recovery key can be received
+    /// in two ways:
+    ///
+    /// 1. Received as a `m.secret.send` to-device event, after a successful
+    ///    interactive verification.
+    /// 2. Imported from secret storage (4S) using the
+    ///    [`SecretStore::import_secrets()`] method.
+    ///
+    /// [`SecretStore::import_secrets()`]: crate::encryption::secret_storage::SecretStore::import_secrets
+    OneShot,
+
+    /// Attempt to download a single room key if an event fails to be decrypted.
+    AfterDecryptionFailure,
+
+    /// Don't download any room keys automatically. The user can manually
+    /// download room keys using the [`Backups::download_room_key()`] methods.
+    Manual,
+}
+
+impl From<BackupDownloadStrategy> for matrix_sdk::encryption::BackupDownloadStrategy {
+    fn from(strat: BackupDownloadStrategy) -> Self {
```

Comment:
nit: can you spell it `strategy`?

Reply:
Thanks, I don't know what came over me! Fixed in cf35c38dad